logging/logging/handlers.py

```diff
@@ -0,0 +1,66 @@
+import os
+import sys
+
+
+def try_remove(fn: str) -> None:
+    """Try to remove a file if it existst."""
+    try:
+        os.remove(fn)
+    except OSError:
+        pass
+
+
+def get_filesize(fn: str) -> int:
+    """Return size of a file."""
+    return os.stat(fn)[6]
+
+
+class RotatingFileHandler:
+    """A rotating file handler like RotatingFileHandler.
+
+    Compatible with CPythons `logging.handlers.RotatingFileHandler` class.
+    """
+
+    def __init__(self, filename, max_bytes=1024, backup_count=3):
```

Comment:
>  Param names are incorrect.

Hm, I don't like the camel case and try to avoid it in my own code. But you are right we try to stay as close to CPython as possible so the variable names should fit. Thanks for fixing.

Reply:
It's well known that CPython's stdlib, written by different people over long time, doesn't have consistency here. But it doesn't matter what we like, if we implement *a well-known API defined not by us*, and that's what we're doing here.